portfolio/src/main/webapp/script.js

```diff
@@ -60,23 +60,35 @@ function getRandomItemFromArray(array) {
 /**
  * Fetch information from the 'data' servlet.
  */
-function fetchFromData() {    
+function fetchFromData() { 
   let commentsEl = document.getElementById("all-comments");
-  fetch('/data').then(response => response.json()).then((comments) => {
-    if (comments.length > 0) {
-      commentsEl.style.visibility = 'visible';
-    }
+  commentsEl.innerHTML = "";
+  const commentNum = document.getElementById("comment-num").value;
+  const commentNumParam = "comments-num";
+  fetch('/data?' + commentNumParam + '=' + commentNum).then(response => response.json()).then((comments) => {
+    commentsEl.style.visibility = (comments.length > 0 ? 'visible' : 'hidden');
     comments.forEach((singleComment) => {
-      commentsEl.appendChild(createListElement(singleComment));
+      commentsEl.appendChild(createTableElement(singleComment));
     });
   });
 }
 
 /** 
- * Creates an <li> element containing text. 
+ * Creates a record in a table. 
  */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createTableElement(comment) {
+  const tableElement = document.createElement('tr');
+  let content = "<td class = 'comment-content'>" + comment.content + "</td>";
```

Comment:
I think you should replace `let` with `const` here as well.

Reply:
done